gravitee-apim-console-webui/src/portal/homepage/homepage.component.ts

```diff
@@ -130,12 +126,16 @@ export class HomepageComponent {
         switchMap((_) => toggleApiCall$),
         tap((updatedPage) => {
           const currentNav = this.portalHomepage()?.navigationItem ?? null;
-          this.portalHomepage.set(
-            // TODO: handle updated navigation item when that feature is implemented
-            currentNav
-              ? { navigationItem: currentNav, content: { id: updatedPage.id, type: 'GRAVITEE_MARKDOWN', content: updatedPage.content } }
-              : null,
-          );
+          this.portalHomepage.update((current) => {
+            return {
+              navigationItem: { ...currentNav, published: updatedPage.published },
+              content: {
+                id: updatedPage.id,
+                type: 'GRAVITEE_MARKDOWN',
+                content: current?.content?.content ?? '',
+              },
+            };
+          });
```

Comment:
I don't think we need to touch the content at all on a publish/unpublish 🤔 WDYT?
```suggestion
          this.portalHomepage.update((current) => {
            return {
              ...current,
              navigationItem: { ...currentNav, published: updatedPage.published },
            };
          });
```